persistence.js

```diff
@@ -180,23 +180,22 @@ MemoryPersistence.prototype.outgoingEnqueueCombi = function (subs, packet, cb) {
 }
 
 function _outgoingEnqueue (sub, packet) {
-  var id = sub.clientId
-  var queue = this._outgoing[id] || []
+  const id = sub.clientId
+  const queue = this._outgoing[id] || []
 
   this._outgoing[id] = queue
-  var p = new Packet(packet)
+  const p = new Packet(packet)
   queue[queue.length] = p
 }
 
 MemoryPersistence.prototype.outgoingUpdate = function (client, packet, cb) {
-  var i
-  var clientId = client.id
-  var outgoing = this._outgoing[clientId] || []
+  const clientId = client.id
+  const outgoing = this._outgoing[clientId] || []
   var temp
 
   this._outgoing[clientId] = outgoing
 
-  for (i = 0; i < outgoing.length; i++) {
+  for (var i = 0; i < outgoing.length; i++) {
```

Comment:
Could you replace all `var i = 0` of for loops with `let i = 0`?

Reply:
@mcollina any concerns?

Reply:
No performance benefit but `let` is block scope so `i` will be defined only inside that `for` loop. It's a good practise

Reply:
@mcollina ping

Reply:
let in loops _was_ slower in Node 10. I think it’s still valid for 12. So, keep it var.

Reply:
> let in loops was slower in Node 10

:disappointed_relieved: didn't know this